docs/sni-limitation.rst

```diff
@@ -0,0 +1,35 @@
+.. _sni-limitation:
+
+Configure SNI TLS Load Balancer
```

Comment:
"Configure an SNI TLS Load Balancer Using the F5 Dashboard for Neutron LBaaS"

Reply:
done